source/_integrations/media_extractor.markdown

```diff
@@ -90,6 +44,19 @@ There is also a service that will directly return the URL in the response of the
 | `url`                  | no       | The URL of the media to extract.                       |
 | `format_query`         | yes      | The query that will be used to select the right media. |
 
+### Format query
+
+The query can be used to select the right media.
+Query examples with explanations:
+
+- **bestvideo**: Best video only stream
+- **best**: Best video + audio stream
```

Comment:
What do you mean by best? High resolution?

Reply:
I think so yes